lib/resources/resources.rb

```diff
@@ -16,10 +16,9 @@ class Barcode < Resource
     attr_accessor :barcode, :id, :product_id, :stamp
   end
 
-  class BarcodeSKU < Resource
+  class BarcodeSku < Resource
```

Comment:
alternatively we could define an abbreviation for "SKU" using Rails' inflector, if the capitalization is messing up autoload